driver_android.go

```diff
@@ -89,15 +97,101 @@ static char* initAudioTrack(uintptr_t java_vm, uintptr_t jni_env,
     return "invalid bitDepthInBytes";
   }
 
-  const jobject tmpAudioTrack =
-      (*env)->NewObject(
-          env, android_media_AudioTrack,
-          (*env)->GetMethodID(env, android_media_AudioTrack, "<init>", "(IIIIII)V"),
-          android_media_AudioManager_STREAM_MUSIC,
-          sampleRate, channel, encoding, bufferSize,
-          android_media_AudioTrack_MODE_STREAM);
-  *audioTrack = (*env)->NewGlobalRef(env, tmpAudioTrack);
-  (*env)->DeleteLocalRef(env, tmpAudioTrack);
+  if (availableSDK >= 24) {
```

Comment:
Add a comment to use FLAG_LOW_LATENCY when the SDK version is >= 24.

Reply:
Added a comment and updated